source/linux/basic/filesystem.rst

```diff
@@ -0,0 +1,188 @@
+	* **.** is a link, that exists in every directory. **It points to the directory, that it is in**.
+	* **..** is a link, that exists in every directory. **It points to the parent directory of the directory it is in**. (e.g. /home/user/.. points to /home)
+	* **~** is everywhere useable. **It points to the home directory of the current user**
+
+
+	Note, that a path can be an absolute or relative. An absolute path starts every time with the root file system (e.g. /home/user/documents). A relative path is relative to the working directory and can even be only a file name (e.g. documents, ../../home)
+
+
+.. hint::
+	Note, that the notation for command arguments is as follows:
+
+	- arguments enclosed with <> are placeholders for required arguments.
+	- arguments enclosed with [] are placeholders for optional arguments.
+
+cd
+^^
+	Full Name:
+		**C** hange **D** irectory
+
+	Description:
+		Changes current working directory
+
+	Example Usage:
+		.. code-block:: bash
+
+			host$ cd /var/lib
+
+		changes working directory to the lib folder that is in the var folder which is in the / folder
+
+		.. code-block:: bash
+
+			host$ cd ./documents
+
+		changes working directory to the documents folder, that is in the current working directory
+
+pwd
+^^^
+	Full Name:
+		**P** rint **W** orking **D** irectory
+
+	Description:
+		Outputs the current working directory
+	Example Usage:
+		.. code-block:: bash
+
+			host$ pwd
+
+ls [path]
+^^^^^^^^^
+	Full Name:
+		**L** ist **D** irectory
+
+	Description:
+		Lists the files and folders in the working directory or in the given path
+	Example Usage:
+		.. code-block:: bash
+
+			host$ ls documents/projects
+		
+		lists the contents of the project folder, thats in the documents folder which is in the working directory
+
+mkdir <path>
+^^^^^^^^^^^^
+	Full Name:  
+		**M** ma **k** e **Dir** ectory
+
+	Description:
+		Creates a Directory
+	Example Usage:
+		.. code-block:: bash
+
+			host$ mkdir documents/projects/newProject
+
+touch <path>
+^^^^^^^^^^^^
+	Description:
+		Creates a file
+	Example Usage:
+		.. code-block:: bash
+
+			host$ touch newFile
+
+		creates the file "newFile" in the working directory
+
+mv <oldPath> <newPath>
+^^^^^^^^^^^^^^^^^^^^^^
+	Full Name:  
+		**M** o **v** e
+
+	Description:
+		Moves files or folders
+	Example Usage:
+		.. code-block:: bash
+
+			host$ mv flash.bin output/flash.bin
+
+rm <path>
+^^^^^^^^^
+	Full Name:
+		**R** e **m** ove
+
+	Description:
+		Removes files or folders
+	Example Usage:
+		.. code-block:: bash
+
+			host$ rm flash.bin
+			or to remove a directory:
+			host$ rm -r ./output
+
+ln -s <path> <linkPath>
+^^^^^^^^^^^^^^^^^^^^^^^
+	Full Name:  
+		**L** i **n** k
+
+	Description:
+		Links files or folders
+	Example Usage:
+		.. code-block:: bash
+
+			host$ host$ ln flash.bin flash_link.bin

```

Comment:
There are to host$ here,
Also above you mention the -s option but your example does not use it. Maybe you can add a description for symbolic links.